app/lib/submission_builder/ty2024/states/nj/documents/nj1040.rb

```diff
@@ -130,6 +130,8 @@ def document
                   if calculated_fields.fetch(:NJ1040_LINE_42) > 0
                     xml.NewJerseyTaxableIncome calculated_fields.fetch(:NJ1040_LINE_42)
                   end
+
+                  xml.ChildDependentCareCredit calculated_fields.fetch(:NJ1040_LINE_64)
```

Comment:
[DUST] In the case where the `federal_child_and_dependent_care_credit` = 0, this will write 0 into the XML. Should we have an if statement that only writes it if it's greater than 0?